mino/minogrpc/addr.go

```diff
@@ -50,6 +53,16 @@ func (a address) GetDialAddress() string {
 	return a.host
 }
 
+// GetHostname parses the address to extract the hostname.
+func (a address) GetHostname() (string, error) {
+	url, err := url.Parse(fmt.Sprintf("//%s", a.host))
+	if err != nil {
+		return "", xerrors.Errorf("malformed address: %v", err)
```

Comment:
```suggestion
		return "", xerrors.Errorf("failed to parse address: %v", err)
```

otherwise you get a: `malformed address: malformed address: ...` error message

Reply:
If forgot to write the test for the hostname, but after doing it, it seems like the error makes sense.